mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java

```diff
@@ -105,7 +105,7 @@
     public static final AttributeName DEFAULT_SOURCE = AttributeName.of(DEFAULT_SOURCE_ATTRIBUTE_NAME);
     public static final AttributeName DEFAULT_RAW_SOURCE = AttributeName.of(DEFAULT_RAW_SOURCE_ATTRIBUTE_NAME);
     public static final AttributeName DEFAULT_DESTINATION = AttributeName.of(DEFAULT_DESTINATION_ATTRIBUTE_NAME);
-    public static final String REPLY_TO_HEADER_NAME = "replyTo";
+    public static final String REPLY_TO_HEADER_NAME = "Reply-To";
```

Comment:
No test broke? Do we miss a unit test?

Reply:
the test uses this constant

Reply:
Tests relying on production constants do have limited interest (eg this issue)

Maybe we should consider putting the raw value in tests instead.